v2/src/modules/tokens/components/TokenDetailsHolder/TokenHolderTable.vue

```diff
@@ -47,36 +44,39 @@
                     :price="tokenPrice"
                 />
             </div>
-            <app-intersect v-if="props.hasMore" @intersect="loadMoreData">
-                <div class="skeleton-box rounded-xl mt-1 my-4" style="height: 24px"></div>
-            </app-intersect>
         </div>
         <!-- End Table Row ERC20 -->
 
         <!-- Table Row NFT -->
-        <v-row v-else-if="!props.initialLoad && tokensNft.length > 0" class="p-ten-top">
-            <v-col v-for="(holder, index) in tokensNft" :key="index" cols="6" sm="4" lg="2" class="p-ten-top">
-                <token-nft-img v-if="!loading" :loading="loadingMeta" :nft="holder" height="240" class="rounded-md"></token-nft-img>
-                <div v-if="!loading" class="d-flex align-center">
-                    <app-address-blockie :address="holder.holder || ''" :size="8" class="mr-1 mr-sm-2" />
-                    <div>
-                        <p>Owned By</p>
-                        <app-transform-hash is-blue is-short :hash="eth.toCheckSum(holder.holder || '')" :link="`/address/${holder.holder}`" />
+        <v-row v-else-if="!props.initialLoad && isNft" class="p-ten-top">
+            <template v-if="!loading && tokensNft.length > 0">
```

Comment:
can you make this `v-if="!props.loading && !props.loadingMeta && tokensNft.length > 0"`